decentraland/unity-renderer
# feat: Add ability to join a public channel

## What does this PR change?
Players can join an existing channel by using the /join name-of-channel command.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/join-public-channel
2. Type `/join <name of a channel>` in the chat.
3. Notice the channel window is opened.
4. Notice the channel has been added in the channels list.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
